docs/guides/payments.rst

```diff
@@ -0,0 +1,226 @@
+.. _adding-payments:
+
+Payments
+========
+
+Integrating new Payment Gateway into Saleor
+-------------------------------------------
+
+We are using a universal flow, that each provider should fulfill, there are
+several methods that should be implemented.
+
+Your changes should live under the
+``saleor.payment.providers.<provider name>``.
+
+.. note::
+
+    After completing those changes your new payment gateway will be available
+    only from the API level. You also need to integrate it into your
```

Comment:
I would say:
After completing those ~~changes~~ steps*,* your new payment gateway will **only** be available from API level. **You will also need to...**